server/dash/dashboards/neighborhood.py

```diff
@@ -1,20 +1,11 @@
-import textwrap
-
-import dash_core_components as dcc
-import dash_html_components as html
 import pandas as pd
```

Comment:
Ah, I just remembered that Piero is updating this file right now. Let's wait until he merges his PR, and then could you pull and update the dash stuff please?